deploy/docker/hail_search/readiness_probe

```diff
@@ -0,0 +1 @@
+curl http://localhost:${HAIL_SEARCH_SERVICE_PORT} | grep -i "html" >& /readiness_probe.log
```

Comment:
k8s conceptual question here.  I'm using an environment variable for the service port here, to mirror what the seqr app is doing.  But I'm actually wondering if we should have this be hard-coded to 8000, which is hard-coded inside the container and also as the `containerPort` [here](https://github.com/broadinstitute/seqr-helm/blob/main/charts/seqr/templates/deployment.yaml#L38).  From my understanding, "service port" is the port the service itself is listening, and that what we really want is the 'containerPort' or the service's 'targetPort' for the readiness probe.

Reply:
FWIW I don't really see any point in mirroring the seqr app, its not like its a particularly great example of using docker and kubernetes. I would go with what seems logical, not whats consistent with seqr